File: ext/no2F2.c
#include <stdio.h>
#include <stdlib.h>

#include "2F2.h"

double hyp2F2(double a1, double a2, double b1, double b2, double z) {
  fprintf(stderr, "Fractional derivatives are not supported!");
  fprintf(stderr, "Please, recompile Multiwfn with its support!");
  exit(1);
  return 0.0;
}

File: ext/no2F2.f90
module no2F2_WIN
contains
  real(8) function hyp2F2(a, b, c, d, z) bind(C, name="hyp2F2")
    real(8), intent(in), value :: a, b, c, d, z
    write(7,*) "Fractional derivatives are not supported!"
    write(7,*) "Please, recompile Multiwfn with its support!"
    error stop
  end function hyp2F2
end module no2F2_WIN